department-of-veterans-affairs/caseflow
# Only add assign to VISN action if FT is enabled

Resolves [CASEFLOW-2588](https://vajira.max.gov/browse/CASEFLOW-2588)

### Description
Hides "Assign to VISN" action unless `visn_predocket_workflow` feature toggle is enabled

### Acceptance Criteria

### Testing Plan
1. As a VHA PO user, visit an appeal that has been assigned by CAMO
2. Ensure the "Assign to VISN" action doesn't show up in the action dropdown.
3. Add the `visn_predocket_workflow` feature toggle and revisit the appeal. The "Assign to VISN" action should show up in the action dropdown.

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1474" alt="Screen Shot 2021-11-22 at 8 37 03 AM" src="https://user-images.githubusercontent.com/640182/142871300-b19e29bf-79ed-402a-bbe1-3a48e39071be.png">|<img width="1460" alt="Screen Shot 2021-11-22 at 8 37 42 AM" src="https://user-images.githubusercontent.com/640182/142871264-60c4dd93-c1ed-44b3-89f8-054320958cbd.png">
